src/pcre2_jit_compile.c

```diff
@@ -8960,7 +8960,7 @@ while (1)
     common->accept_label = save_accept_label;
     common->positive_assertion_quit = save_positive_assertion_quit;
     common->accept = save_accept;
-    return NULL;
+    goto end;
```

Comment:
Nice catch! I would use the `return cc + 1 + LINK_SIZE;` everywhere.

Reply:
I am interpreting that as, "don't use the goto".

If you meant, "change the returns in other places as well" let me know where.

Reply:
I did _yet another_ attempt at fixing the JIT segfault. I realised, just before merging this, that `return cc + 1 + LINK_SIZE` is not correct. The return statement is inside a loop which increments `cc`, so that's not the correct pointer to return!

It basically does have to return NULL in this instance... so I updated the call site with an error check.